docroot/modules/custom/foia_webform/src/FoiaSubmissionServiceApi.php

```diff
@@ -371,13 +392,44 @@ protected function getAttachmentData(array $files) {
     if (!empty($files)) {
       foreach ($files as $fid) {
         $currentFile = File::load($fid);
-        $base64 = base64_encode(file_get_contents($currentFile->getFileUri()));
-        $fileContents[] = [
-          'content_type' => $currentFile->getMimeType(),
-          'filedata' => $base64,
-          'filename' => $currentFile->getFilename(),
-          'filesize' => (int) $currentFile->getSize(),
-        ];
+        $virusScanStatus = $currentFile->get('field_virus_scan_status')->value;
+        if ($virusScanStatus !== 'virus') {
+          $base64 = base64_encode(file_get_contents($currentFile->getFileUri()));
+          $fileContents[] = [
+            'content_type' => $currentFile->getMimeType(),
+            'filedata' => $base64,
+            'filename' => $currentFile->getFilename(),
+            'filesize' => (int) $currentFile->getSize(),
+          ];
+        }
```

Comment:
If the $virusScanStatus *is* `virus`, should we or do we need to set $fileContents to `NULL` or something else?